experimental-examples/tina-cloud-starter/.tina/config.ts

```diff
@@ -15,9 +15,9 @@ export default defineConfig({
   clientId: "foobar",
   token: "foo",
   media: {
-    tina: {
-      publicFolder: "public",
-      mediaRoot: "",
+    loadCustomStore: async () => {
```

Comment:
did you intend to leave these changes?

Reply:
I can remove them. (it was just used for testing)